src/variation/translators/amplification.py

```diff
@@ -52,7 +53,7 @@ async def translate(
         if priority_seq_loc:
             vrs_cx = models.CopyNumberChange(
                 location=models.SequenceLocation(**priority_seq_loc),
-                copyChange=models.CopyChange.EFO_0030072,
+                copyChange=MappableConcept(primaryCode=models.CopyChange.EFO_0030072),
```

Comment:
do we need to be providing a `mapping` with an `exactMatch` to this EFO term as well?

Reply:
Ah yes, thank you. Forgot to update with this change

Reply:
@jsstevenson I thought about putting this in VRS-Python, but since there's no guidance / standard at the moment on what values to use, I'm putting it in Variation Normalizer.